web_src/js/index.js

```diff
@@ -65,14 +65,15 @@ function initEditPreviewTab($form) {
     $previewTab.on('click', function () {
       const $this = $(this);
       let context = `${$this.data('context')}/`;
+      const mode = this.data('markdown-mode') || 'comment';
```

Comment:
I guess this should actually be `$this.data`? Or just use `this.dataset.markdownMode`.

Reply:
Or maybe you need to select for the `textarea` that has the attribute as `this` should refer the tab here, not the textarea.

Reply:
It genuinely worked in testing! I've put the gfm mode on the preview tab.

Reply:
Looking good now.